openstack/sharedfilesystems/v2/shares/results.go

```diff
@@ -144,3 +144,36 @@ func (r GetExportLocationsResult) Extract() ([]ExportLocation, error) {
 	err := r.ExtractInto(&s)
 	return s.ExportLocations, err
 }
+
+// AccessRight contains all information associated with an OpenStack share
+// Grant Access Response
+type AccessRight struct {
+	// The UUID of the share to which you are granted or denied access.
+	ShareID string `json:"share_id"`
+	// The access rule type that can be "ip", "cert" or "user".
+	AccessType string `json:"access_type,omitempty"`
```

Comment:
> Does omitempty provide any use here?

No; I think `omitempty` is only relevant for encoding. But it doesn't hurt either, so... meh.